game/graphics/opengl_renderer/OpenGLRenderer.cpp

```diff
@@ -1304,7 +1304,8 @@ void OpenGLRenderer::dispatch_buckets_jak1(DmaFollower dma,
     m_category_times[(int)m_bucket_categories[bucket_id]] += bucket_prof.get_elapsed_time();
 
     // hack to draw the collision mesh in the middle the drawing
-    if (bucket_id == 31 - 1 && Gfx::g_global_settings.collision_enable) {
+    if (bucket_id == (int)jak1::BucketId::MERC_PRIS_LEVEL0 &&
+        Gfx::g_global_settings.collision_enable) {
```

Comment:
I think moving this here will break shadows on the collision geometry but I'm not sure. What if you put `GENERIC_ALPHA` as the bucket instead?

Reply:
![image](https://github.com/user-attachments/assets/d9ce0e60-2b6f-4401-9369-33c009b7554c)
I binary searched the enum and found an entry that "worked" - I did a test through hub 1 and didn't notice any issues after my change but I can do another test and pay attention to shadows specifically.